core/steam_client.gd

```diff
@@ -80,6 +86,17 @@ func bootstrap() -> void:
 	bootstrap_finished.emit()
 
 
+## Waits until the local machine can resolve valvesoftware.com
+func wait_for_network() -> void:
+	var network_state := network_manager.NM_STATE_UNKNOWN
+	while network_state != network_manager.NM_STATE_CONNECTED_GLOBAL:
```

Comment:
This seems bad for a device not connected to the Internet?

Reply:
I think it just waits until a network is connected before starting steamcmd. Should be fine, even it may wait forever ...